src/components/organisms/FetchList/FetchList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import { List } from '../List/List';
+
+export const FetchList = () => {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
Bardzo często stosuje się podejscie gdzie jest stan "isLoading" - wtedy ustawia się na true dopiero, gdy zaczynamy pobierać dane, a na false gdy mamy odpowiedź. 

Podejscie z "isLoaded" przypomina mi trochę tą bibliotekę https://swr.vercel.app/ 
Generalnie ten projekt jest stosunkowo mały zatem to jak zrobiłas fetch jest zupełnie poprawne. W dużych projektach są wrappery na zapytania i często wykorzystuje się na przykład axios (troszkę bardziej dopakowany fetch) + bibliotekę pod reacta z różnymi gotowymi rozwiązaniami jak własnie SWR lub React Query. 

Jesli chcesz możesz dla testu spróbować któres z nich wykorzystać :)